frontend/packages/helm-plugin/src/components/details-page/HelmReleaseDetails.tsx

```diff
@@ -154,7 +153,7 @@ const HelmReleaseDetails: React.FC<HelmReleaseDetailsProps> = ({ secret, match }
     return () => {
       ignore = true;
     };
-  }, [helmReleaseName, namespace]);
+  }, [helmReleaseName, namespace, secret.data]);
```

Comment:
`secret.data` isn't used in this effect. Can you explain why this is here and if necessary add a comment in the code.